### PR TITLE
Enable CI

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: tests/integration/target-prefixes.network [hunk 1]
```diff
@@ -1 +1 @@
-netconf
\ No newline at end of file
+yang
```
Comment (ganeshrn) [inline tests/integration/target-prefixes.network]: This change is not required and the file can be empty Refer https://github.com/ansible/ansible-zuul-jobs/blob/10fd68e938f5a4b369b6f60b623dc467cda26bf5/zuul.d/jobs.yaml#L1918

File: zuul.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+- project:
+    templates:
+      - ansible-collections-community-yang-juniper-junos
```
Comment (ganeshrn) [inline zuul.yaml]: add `ansible-collections-community-yang-cisco-iosxr` for iosxr

**Review by ganeshrn — COMMENTED** [review]
renaming test folder with `yang_` is not required as mentioned in the review comment

File: tests/integration/targets/prepare_iosxr_tests/aliases [hunk 3]
```diff
@@ -0,0 +1 @@
+network/iosxr
```
Comment (ganeshrn) [inline tests/integration/targets/prepare_iosxr_tests/aliases]: change this to `network/yang` after adding the `yang` group in inventory template Refer
1) https://github.com/ansible/ansible-zuul-jobs/blob/1586f4e2aad2a211c7c7b196a937fd26d8c86abd/roles/ansible-test-inventory/templates/inventory-iosxr.j2#L7

2) https://github.com/ansible/ansible-zuul-jobs/blob/1586f4e2aad2a211c7c7b196a937fd26d8c86abd/roles/ansible-test-inventory/templates/inventory-iosxr.j2#L7

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
Already fixed by other PR

**Pull request closed without merging**